include/pops/simulation.hpp

```diff
@@ -215,11 +215,15 @@ class Simulation
     }
 
     /** Moves hosts from one location to another
+     *
+     * @note Note that unlike the other functions, here, *total_hosts*,
+     * i.e., number of hosts is required, not number of all hosts
+     * and non-host individuals.
      *
      * @param infected Currently infected hosts
      * @param susceptible Currently susceptible hosts
      * @param mortality_tracker Hosts that are infected at a specific time step
-     * @param total_hosts total number of hosts
+     * @param total_hosts Total number of hosts
```

Comment:
total_populations

Reply:
We either need to change it everywhere and just make it explicitly clear (which you have done) that total_populations can only include hosts when movement is used. Otherwise, we need to use 2 parameters (total_hosts and total_populations) just to support this aspect of movements in the near-term until the movements class is redone in #87. I am open to either option just think changing it is easier for handling the data in R and Grass but it does put the onious on the user to pay attention to the documentation here.

Reply:
Actually, this would be a good first segway into handling the muli-host dynamics with host competency and susceptibility.

Reply:
I actually meant for this to be total_hosts. My thinking here was to keep the naming as whatever it actually is, so total_hosts. The parameter names are not part of API in C++, so they are more documentation and for clarity of the code. Also, this is internal to core, so it should not be visible to rpops and r.pops.spread. Hence, total_hosts here, total_populations elsewhere. I think it should be changed everywhere in rpops and r.pops.spread since what I'm saying does not apply there. On the other hand, the point was to document it well which includes avoiding confusion which is perhaps what I created.

Reply:
That makes complete sense now! It makes it clear when we go to update the movements class that this is the assumption here. Good call. Merge when ready!